tests/unittests/ThreadPoolExecutorTest.cpp

```diff
@@ -355,7 +366,8 @@ class ExecutorTestBuilder final {
     }
 
     // Iterate through inputs and outputs and:
-    // 1) Create Placeholders and Tensors for inputs/output names that have not
+    // 1) Create Placeholders and Tensors for inputs/output names that have
+    // not
```

Comment:
Weird line break here

Reply:
Yeah my auto clang-formatter did it, but I'm wondering if I might have missed a `}` when working and it reformatted based on bad indentation? I'll look into it.